modules/eventing/pages/eventing-examples-high-risk.adoc

```diff
@@ -2,129 +2,671 @@
+
+2020-03-22T14:24:25.496-07:00 [INFO] "transaction:tx-1505315650-403 rates " {"type":"exchange_rates","erid":"er-2018-09-14","to_USD":{"CAD":1.3008811703,"INR":71.8162374882,"EUR":0.8555051758,"USD":1,"SGD":1.3698348875,"GBP":0.7633501583,"CNY":6.8543074686,"AUD":1.3910514159}} 
+
+2020-03-22T14:24:25.496-07:00 [INFO] "transaction:tx-1505315650-403 1 doc.amount        5383.36, card_limit        9500.00" 
+
+2020-03-22T14:24:25.496-07:00 [INFO] "transaction:tx-1505315650-403 2 trxn_currency         GBP, card_currency         USD" 
+
+2020-03-22T14:24:25.496-07:00 [INFO] "transaction:tx-1505315650-403 3 trxn_2_USD       0.763350, card_2_USD       1.000000" 
+
+2020-03-22T14:24:25.496-07:00 [INFO] "transaction:tx-1505315650-403 4 trxn_amount_USD   7052.28, card_thresh_USD   9500.00" 
+
+2020-03-22T14:24:25.496-07:00 [INFO] "transaction:tx-1505315650-403 *** High Risk Txn: currency mismatch card: USD != txn: GBP" 
 ----
 +
-image::highrisk_05_n1ql_query.png[,%100]
+Above we see that for one mutation the Function *high_risk_txns* printed the following log messages:
+** The transaction document 'doc' that mutated (what you just changed)
+** The 'card' or credit card that the transaction was made with was looked up via the key _'card:' + doc.card_ from the bucket alias  register.
+** A daily 'rates' or exchange rates for the date of the transaction which was looked up via the key _'exchange_rates:er-' + (doc.date).substr(0, 10))_ from the bucket alias register.
+** Next a bit of information (4 lines) was emitted  to show what was done in order to make the final decision.
+** Finally, the last log line is the decision line if the transaction is considered 'High Risk'.
+The amount of information at this verbosity level would be excessive for millions of documents, however it can be very handy for developing and debugging Eventing handler JavaScript logic.
+
+. From the *Eventing* screen, click *Undeploy*.
++
+image::high_risk_txns_10_pause.png[,%100]
+** In the *Confirm Undeploy Function* dialog, Click *Undeploy Function*.
++
+image::high_risk_txns_11_undeploy_confirm.png[,346]
+
+. The Eventing function will be undeployed within a few seconds, this concludes the Example.
+
+. Cleanup, go to the Eventing portion of the UI and Undeploy the Function *high_risk_txns*, this will remove the 1024 documents (64 for macOS) from the 'metadata' bucket (in the Bucket view of the UI). Remember you may only delete the 'metadata' bucket if there are no deployed Eventing functions. Delete the *'register'* and *'review'* buckets and also the empty *'metadata*' bucket.
```

Comment:
```suggestion
. Undeploying the function *high_risk_txns* removes the 1024 documents (64 for macOS) from the *metadata* bucket, as you can see in the Bucket view of the UI. Remember you may only delete the *metadata* bucket if there are no deployed Eventing functions. To clean up, delete the *register* and *review* buckets and also the empty *metadata* bucket.
```

Reply:
A prefer to something starting  with the word 'Cleanup' it is consistent with the final steps in my other Examples.  I have a reason for the use of the word 'Cleanup' here as I plan to reference this in blog and stress that the final step 'Cleanup' be skipped.

===== maybe the Example can end with the following ====

. Cleanup involves undeploying and deleting the function and then removing the three buckets you created. This concludes the Example.

** From the *Eventing* screen, click *Undeploy*.
+
image::high_risk_txns_10_pause.png[,%100]
+
** In the *Confirm Undeploy Function* dialog, Click *Undeploy Function*.
+
image::high_risk_txns_11_undeploy_confirm.png[,346]

** The Eventing function will be undeployed within a few seconds. 

** Undeploying the function *high_risk_txns* removes the 1024 documents (64 for macOS) from the *metadata* bucket, as can be seen in the Bucket view of the UI. Remember you may only delete the *metadata* bucket if there are no currently deployed Eventing functions.

** To remove all of the artifacts: 1) delete the *high_risk_txns* function 2) delete the three buckets *register*, *review*, and *metadata* (the later *metadata* should be empty).